services/api/src/middlewares/error-handler.js

```diff
@@ -4,25 +4,15 @@ module.exports = async (ctx, next) => {
   try {
     await next();
   } catch (err) {
-    const errorStatus = Number.isInteger(err.status) ? err.status : 500;
-    ctx.status = errorStatus;
-    if (err.isJoi) {
```

Comment:
i think for sure you want to check if this joi error and then apply logic, there can be my types of error throws, e.g. e.details might not contain what you think.

Reply:
That's a good point, will have a look